tests/ui/needless_late_init.stderr

```diff
@@ -0,0 +1,108 @@
+error: unneeded late initalization
+  --> $DIR/needless_late_init.rs:4:5
+   |
+LL |     let a;
+   |     ^^^^^^
+   |
+   = note: `-D clippy::needless-late-init` implied by `-D warnings`
+help: declare `a` here
+   |
+LL |     let a = match n {
+   |     +++++++
+help: remove the assignments from the match arms
+   |
+LL ~         1 => "one",
+LL |         _ => {
+LL ~             "two"
```

Comment:
Another problem I had was with mutlispan_suggest, the suggestions are a little bare and using _verbose doesn't seem to change it. As well as running into https://github.com/rust-lang/rustfix/issues/141

Is there a way to have a regular suggestion appear just without the help label repeated?